README.md

```diff
@@ -1,6 +1,6 @@
 # openfunction
 
-![Version: 0.2.0](https://img.shields.io/badge/Version-0.2.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.7.0](https://img.shields.io/badge/AppVersion-0.7.0-informational?style=flat-square)
+![Version: 0.3.0-rc.0](https://img.shields.io/badge/Version-0.3.0--rc.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.0-rc.0](https://img.shields.io/badge/AppVersion-0.8.0--rc.0-informational?style=flat-square)
```

Comment:
`Version-0.3.0--rc.0`
`AppVersion-0.8.0--rc.0`

there is an extra `-` , is that ok?

Reply:
This extra `-` is needed.

Reply:
https://img.shields.io/badge/AppVersion-0.8.0--rc.0-informational?style=flat-square